github-activity-tracker/backend/.env

```
@@ -0,0 +1,7 @@
+RDS_HOST=your-rds-endpoint
+RDS_PORT=5432
+RDS_DATABASE=github_data
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RDS_DATABASE key should go before the RDS_HOST key

(UnorderedKey)